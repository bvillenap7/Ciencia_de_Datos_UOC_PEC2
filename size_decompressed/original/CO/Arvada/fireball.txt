I was on my way to school Thursday morning, around 6AM. As I looked out the windshield, I saw a huge white ball of light with a tail, l;2014-12-11 06:40:00;1 minute;I was on my way to school Thursday morning, around 6AM.  As I looked out the windshield, I saw a huge white ball of light with a tail, like a comet.  Several seconds later, the ball of light accelerated its speed and literally vanished into mid-air, almost looking like it entered into a invisible portal of some sort!   I freaked out and asked everyone in the vehicle if they saw the comet-like object disappear into mid air, and of course no one did besides myself. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
Orange fire low in sky.;2016-05-07 22:40:00;?;Orange fire low in sky. I was driving west on I-70 and I saw an orange fire in the sky.  It was the shape of a candle's fire, round at the bottom and tear shaped at the top.  I only saw it a few moments.;39.8289982352941;-105.110888235294
