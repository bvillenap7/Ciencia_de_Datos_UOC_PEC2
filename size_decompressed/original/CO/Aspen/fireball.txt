Red. White, green flashing lights in the sky;2011-03-13 23:00:00;?;Red. White, green flashing lights in the sky I saw several stars as I was looking up into the night.  Then I noticed the different colors flashing. Green, white, red, maybe some yellow.  I realized they were way too high in the sky to be planes, so I'm reporting it.  Strange thing is. I don't live in Aspen, and I don't believe in UFOs.  What I saw is unexplainable. Therefore . I'm reporting it. ((NUFORC Note:  Witness indicates that the date of the sighting is approximate.  PD));39.1977833333333;-106.825375
