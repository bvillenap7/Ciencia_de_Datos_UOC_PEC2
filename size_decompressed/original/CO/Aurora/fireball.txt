Orange fireball like seen in Aurora north of Buckley AFB at 0300.Duration of 10 minutes,Then dissappears.Recorded on camera!;2001-07-17 03:00:00;10MINUTES;Orange fireball like seen in Aurora north of Buckley AFB at 0300.Duration of 10 minutes,Then dissappears.Recorded on camera!;39.6695444272445;-104.803991486068
Aurora-sees another fireball in the skies of Colrado!!! A truck driver catches the  fireball on his truck mounted camerra.Local Colrado;2002-11-28 20:30:00;60 seconds;Aurora-sees another fireball in the skies of Colrado!!! A truck driver catches the  fireball on his truck mounted camerra.Local Colrado residents from all over the state get a eye view of the bright fireball.This is the 5th one in only a two month period. With a recent meteor shower this month,people are wondering is the big one on it's  way?? I hope not! Believe it or not,Colradans get once more a shower of meteorites...yet no one has found where the objects crashed into the southern parts of Colrado and New Mexico.This story was air on news9 and in CO.NEWS PAPERS ON THE 29TH OF NOVEMBER YEAR 2002 TO THE DATE!;39.6695444272445;-104.803991486068
green meteor;2004-07-07 01:45:00;10 seconds;green meteor Some friends and I were watching a movie in our living room when my one friend shouted out oh my god what is that?  My other friend and I looked out the window to see a huge brite green like streak accross the sky, it almost resembled a meteor.  It only lasted a few seconds but we all got a really good look of the object. It was going from West to East, and was in sight for a maximum of 10 seconds, and then dissapeared over the horizon.  The event occured at approximately 1:45am MDT over Aurora, Colorado. ((NUFORC Note:  Witness elects to remain totally anonymous.  PD));39.6695444272445;-104.803991486068
Amber Orb hovering over the skies in Aurora Colorado.;2014-03-19 00:40:00;10 minutes;Amber Orb hovering over the skies in Aurora Colorado. While driving home, I saw a bright amber light hovering very high in the southern sky behind some houses.  It must have been a few miles away.  The strange thing was that the object wasn't moving across the sky (like a plane would), but seemed to be fixed.  I could tell it wasn't a star because of the way it kind of morphed, going from solid to fireball shape, and because of the color being solid amber/white.   Luckily I had my camera with me and filmed the event. The entire object, not just part of it, flashed a few times.  At one point, I reached down to look at my cell phone to check the time, and when I looked back, the object was gone.  I drove around the neighborhood trying to find it again, but it was gone. Here is a link to the video I uploaded on youtube:  http://youtu.be/FCA95u123XE;39.6695444272445;-104.803991486068
5 Bright lights seen.;2014-06-05 22:30:00;15 minutes;5 Bright lights seen. 5 bright lights were seen heading west all within a few minutes of each other.  Images would not show up on camera for some reason.  Thought they were meteors, but no news on them, and have never seen planes go that way. ((NUFORC Note:  A second telephoned report from other witnesses in Aurora, CO, was received on the morning of Friday, June 6th.  It appears that both groups witnessed the same cluster of objects.  We are awaiting those other reports from the second set of witnesses, as we post the database on June 6th, 2014.  PD));39.6695444272445;-104.803991486068
Six orange crafts flew over our house late Friday night, one at a time.;2015-05-30 22:30:00;12 minutes;Six orange crafts flew over our house late Friday night, one at a time. On May 30, 2015, two of my daughters, myself, and their dad saw 6 orange crafts fly directly over our house.  One at a time.  They were so bright they looked like fireballs.   We took photos using our cell phones and I had a camera in the car out front, too, so I got some shaky shots with that as well.   They came from south to north approximately 2-4 minutes apart..making no sound at all. They also appeared to be flying lower than a plane would.;39.6695444272445;-104.803991486068
A diamond on fire came straight up and moved sideways very quickly;2016-02-06 19:35:00;30 seconds;A diamond on fire came straight up and moved sideways very quickly. I was outside in the driveway smoking a cigarette and I seen the diamond shaped fireball come straight up  from behind the neighbors houses in the distance from the northeast and it moved sideways to the south towards parker very fast and it looked like it was lowering as it started to get out of my sight because  I could  not see it over the houses at the end of the block anymore it was more orange then red and it had a sparkle like dot at the tip of the bottom it scared me so I tossed my cigarette and went in the house it change my whole outlook on ufo's or other life forms I have never seen anything move horizontally or diagnally that smooth and fast with no sound coming from it!;39.6695444272445;-104.803991486068
I saw a fast moving flash of light at first just a flash then added red light moving across the sky east to west while looking south.;2016-10-27 19:00:00;15 seconds;I saw a fast moving flash of light at first just a flash then added red light moving across the sky east to west while looking south. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.6695444272445;-104.803991486068
I was fishing on the South side of Cherry Creek State Park and there was very little illumination for lights, it was dark and excellent;2019-09-01 23:00:00;3 minutes;I was fishing on the South side of Cherry Creek State Park and there was very little illumination for lights, it was dark and excellent for viewing stars in the night sky.   I was looking up at the sky and thought I seen a red dwarf star, then I realized it was moving North at a very high rate of speed.  In a matter of seconds it was ok I sight.   Maybe a minute later I seen another round orange object move over head at the same rate of speed, almost the same trajectory, then a moment later a third object followed the previous two.  They maintained a straight flight path and was very high overhead one second and out of sight a few seconds later.  Awesome experience!;39.6695444272445;-104.803991486068
