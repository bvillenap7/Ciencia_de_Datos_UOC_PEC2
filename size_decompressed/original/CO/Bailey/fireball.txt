Streak of bright blue/white light flashed vertically downward from @40deg to horizon.  Seen @20degs. SSW in clear moonlit sky.;1998-11-01 22:00:00;<2 secs.;Streak of bright blue/white light flashed vertically downward from @40deg to horizon.  Seen @20degs. SSW in clear moonlit sky. I was watching a movie in my livingroom @ 22:00 when I noticed out my window a very bright bluish-white streak of light to 20 degrees SSW at about 40 degrees angle up and moving vertically down to the horizon where it dissappeared from view.  It lasted less than 2 seconds and due to the angle of travel I ran outside thinking it may have been a meteor which could have hit or come close to it.  Though I was not looking directly at the event, it did appear to have a tail of some sort and seemed to be relatively close to my geographic location.  It was a cloudless, moonlit night and visibility was excellent from my home at 9200 feet elevation.  If further contact with me is necessary please contact me via the information below.  I'm curious if it was witnessed by anyone else.  Thank You...;39.4497;-105.4692
