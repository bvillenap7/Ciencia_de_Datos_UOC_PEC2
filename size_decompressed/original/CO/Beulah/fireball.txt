My guess Top Secret US Military craft perhaps connected to the Secret Space Program.;2019-02-02 18:00:00;2 hours;My guess Top Secret US Military craft perhaps connected to the Secret Space Program. Two very large reddish orange fireballs and a number of white spheres.  Strange sounds like an organ.  First thought they were jets, but that wasn't that case at all.;38.087;-104.9698
