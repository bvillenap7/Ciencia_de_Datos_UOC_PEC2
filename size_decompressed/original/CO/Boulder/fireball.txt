While walking south a large comet like object swept out of the sky and turned horizonel at about a thousand feet from the ground and he;1981-05-01 18:30:00;3 seconds;While walking south a large comet like object swept out of the sky and turned horizonel at about a thousand feet from the ground and headed west disappearing at the point it would have hit the mountain. While walking south a large comet like object swept out of the sky and turned horizonel at about a thousand feet from the ground and headed west disappearing at the point it would have hit the mountain. The object looked like what I thought for many years was a metiorite. It had a hot molten look to it with a firey like tail. I was amazed when it didn't hit the mountain. It flew the length of the Boulder Mall, a very busy place and there was never any mention of it so I just kept quiet. Until I learned that these fire balls have been reported as possiblely vehicles. The point at which it disappeared would be Pearl St. at the base of boulder canyon. The Ball seemed to have a crator like surface.;40.0217727272726;-105.258365102639
01/11/01 fireball over western skies was different color and headed in a different direction.;2001-01-11 21:30:00;5sec;01/11/01 fireball over western skies was different color and headed in a different direction. Just an addendum to the coverage from the AP of the fireball seen on Thursday evening. The report from the AP states The helicopter crew spotted the fireball while flying about 30 miles east of Greeley. The light came from the east, around the Nebraska-Colorado border. But what we witnessed (just happened to have just reached the top of Bear Peak and were looking out across the plains toward Denver) was definitely traveling east and came from the west along a trajectory more like Salt Lake/Denver than Nebraska/Colorado. The glow from the illuminated gases around the object were bright enough to light up my face and allow one of my friends to realize and turn around and catch the end of the spectacle. Also, the AP story states that the observers in Sydney saw a blue fireball. Our observations were definitely of a bright green fireball. Not sure the exact length. I'm guessing around five seconds. Very amazing.;40.0217727272726;-105.258365102639
Boulder, CO, Fireball;2012-11-04 01:30:00;3 seconds;Boulder CO Fireball I was outside on my deck looked up to witness an orange fireball.   It was descending, heading from west to east.  The object was silent. ((NUFORC Note:  We spoke via telephone with this witness, and we found him to be unusually eloquent and sober-minded in his description of the event.  We suspect that the correct time should read 00:44 hrs., but we cannot be certain.  PD));40.0217727272726;-105.258365102639
Green Orb leaving slime trail over Boulder, Colorado 2015.;2015-11-01 19:00:00;15 seconds;Green Orb leaving slime trail over Boulder, Colorado 2015. coming back from best buy with my roommate and we spot a large green orb floating to our right.  The orb left a green slime trail in the sky and was traveling at around 100 Mph.  It then stopped and hovered for 5 seconds until it shot off into the distance in milliseconds.  Two other cars were nearby and stopped to ask us if we just saw that.;40.0217727272726;-105.258365102639
I was on my way home from Bar Louie in Wesrminiter heading down rt. 36 to Boulder. About 4 minutes to Boulder I see a big, bright orang;2019-07-05 02:23:00;2 minutes;I was on my way home from Bar Louie in Wesrminiter heading down rt. 36 to Boulder. About 4 minutes to Boulder I see a big, bright orange, fire ball floating low in the sky. I literally put my hazard lights on and pulled over on the major highway to get out and look at it. I tried taking a video and I thought it was going successful until I got back into my car and the video was messed up. I was shook and excited by the experience. The fireball moved quickly and was very close the the land-comparitively to any air craft. It also made absolutely no noise.;40.0217727272726;-105.258365102639
