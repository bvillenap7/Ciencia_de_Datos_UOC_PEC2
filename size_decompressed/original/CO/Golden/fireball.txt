Green fireball went across the sky and turned orange, then disappeared. It was too large to be a comet, about the size of 5+ airplanes.;2020-02-07 21:50:00;3 seconds;Green fireball went across the sky and turned orange, then disappeared.  It was too large to be a comet, about the size of 5+ airplanes.;39.7598265306123;-105.220002040816
